### DISCUSSION TITLE
p/ef26175b98af43ccaf918296a739ded5/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#7] [Announcements]
# p/ef26175b98af43ccaf918296a739ded5/

今天在社区聊天群友说起 react 状态管理器时突然想起来了一篇旧文 2016 年里做前端是怎样一种体验，所以又模仿它跟着最近的新的技术写了一点。 问：最近我开始了一个新的 web 项目，不过老实说我已经好久没碰过这方面的代码了，之前只写过 vue2。听说前端的技术栈已经发生了极大的变革，不知道你现在是不是仍然处于最前沿的开发者阵列？答: 是的，自从 hooks 出来之后，web 框架又发生了很多

https://blog.rxliuli.com/p/ef26175b98af43ccaf918296a739ded5/

**Comment by jw-12138** [comment]
感觉现在的前端环境，激进得有点不健康🤣

**Reply by rxliuli** [reply]
不一直都是这样？(～￣▽￣)～
